lib/src/play_store_search_api.dart

```diff
@@ -99,15 +99,32 @@ class PlayStoreResults {
   /// Returns field releaseNotes from Play Store results. When there are no
   /// release notes, the main app description is used.
   static String? releaseNotes(Document response) {
+    RegExp releaseNotesSpan = RegExp(r'>(.*?)</span>');
```

Comment:
Does this regex need to be compiled on each call to `releaseNotes` or could it be a class const?

Reply:
I moved it as a static variable outside this method